docs/apis-clients/working-with-apis-tools.md

```diff
@@ -0,0 +1,89 @@
+---
+id: working-with-apis-tools
+title: "Working with APIs & tools"
+sidebar_label: "Working with APIs & tools"
+description: "Interact programmatically with Camunda Platform 8 using official Zeebe client libraries and APIs."
+---
+
+import DocCardList from '@theme/DocCardList';
+
+This section steps through two concepts for integration:
+
+- **Control your Camunda Platform 8 process automation** by [deploying processes](/components/modeler/web-modeler/save-and-deploy.md), [starting process instances](/components/modeler/web-modeler/start-instance.md), [activating jobs](/components/concepts/job-workers.md), and more using supplemental and community-maintained **Zeebe client libraries**.
+- **Interact with the Camunda Platform 8 ecosystem** by learning about [Camunda Components](/components/components-overview.md) and their APIs to communicate with your cluster, search, get and change data, create Cloud API clients, and more.
+
+:::note
+You're permitted to use these web apps and APIs for free with the Free Edition in non-production environments. To use the software in production, [purchase the Camunda Platform Enterprise Edition](https://camunda.com/products/cloud/camunda-cloud-enterprise-contact/). Read more in our [licensing](../reference/licenses.md) documentation.
+:::
+
+## Deploy processes, start process instances, and more using Zeebe client libraries
+
+Clients allow applications to do the following:
+
+- Deploy processes.
+- Start and cancel process instances.
+- Activate jobs, work on those jobs, and subsequently complete or fail jobs.
+- Publish messages.
+- Update process instance variables and resolve incidents.
+
+The official clients mentioned below interact with [Zeebe](/components/zeebe/zeebe-overview.md), the workflow engine integrated into Camunda Platform 8. Clients connect to Camunda Platform 8 via [gRPC](https://grpc.io), a high-performance, open source, and universal RPC protocol.
+
+Camunda Platform 8 provides several official clients based on this API. Official clients have been developed and tested by Camunda. They also add convenience functions (e.g. thread handling for job workers) on top of the core API.
+
+### Official Zeebe clients
+
+Official clients have been developed and tested by Camunda. They also add convenience functions (e.g. thread handling for job workers) on top of the core API.
+
+<DocCardList items={[{type:"link", href:"/docs/apis-clients/java-client/", label: "Java client", docId:"apis-clients/java-client/index"},
+{
+type:"link", href:"/docs/apis-clients/go-client/go-get-started/", label: "Go client", docId:"apis-clients/go-client/index"
+},
+{
+type:"link", href:"/docs/apis-clients/cli-client/index", label: "CLI client", docId:"apis-clients/cli-client/index"
+}
+]}/>
```

Comment:
One thing you and I noted when pairing, which I want to posterize here, is that we don't like the fact that the objects we pass into `<DocCardList>` include a URL that is locked to one specific version. We didn't do any investigation into whether we could use the `$docs$` and `$optimize$` helpers here to prevent that, but we should iterate on that also -- maybe as part of the work we do to customize the `<DocCard>` and `<DocCardList>` components, because that's where we'd have to do it if the helpers don't work.